v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -138,5 +148,55 @@ def requires(self) -> ReferenceDataset:
             RAW_DATASET_PATH: 'gs://seqr-reference-data/GRCh38/TopMed/bravo-dbsnp-all.vcf.gz',
         },
     },
+    ReferenceDataset.hgmd: {
+        ReferenceGenome.GRCh37: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://seqr-reference-data-private/GRCh37/HGMD/HGMD_Pro_2023.1_hg19.vcf.gz',
+            ENUMS: hgmd.ENUMS,
+        },
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://seqr-reference-data-private/GRCh38/HGMD/HGMD_Pro_2023.1_hg38.vcf.gz',
+            ENUMS: hgmd.ENUMS,
+        },
+    },
+    ReferenceDataset.gnomad_exomes: {
+        ReferenceGenome.GRCh37: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/2.1.1/ht/exomes/gnomad.exomes.r2.1.1.sites.ht',
+        },
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/4.1/ht/exomes/gnomad.exomes.v4.1.sites.ht',
+        },
+    },
+    ReferenceDataset.gnomad_genomes: {
+        ReferenceGenome.GRCh37: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/2.1.1/ht/genomes/gnomad.genomes.r2.1.1.sites.ht',
+        },
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/4.1/ht/genomes/gnomad.genomes.v4.1.sites.ht',
+        },
+    },
+    ReferenceDataset.gnomad_qc: {
```

Comment:
This looks identical to the `gnomad_genomes` config so it might be good to do what we did for `clinvar_path` below and do 
```
CONFIG[ReferenceDatasetQuery.gnomad_qc] = CONFIG[ReferenceDataset.gnomad_genomes]
```

Reply:
good idea, I did this with `gnomad_exomes` and `gnomad_genomes`

Reply:
`gnomad_qc` config being identical to gnomad_genomes was a typo when this comment was made so I'll resolve